src/DependencyInjection/Compiler/AddResourcesPathsPass.php

```diff
@@ -41,7 +41,7 @@ private function getResourcesPath(ContainerBuilder $container)
         $rootDir = dirname($container->getParameter('kernel.root_dir'));
 
         foreach ($container->getParameter('kernel.bundles') as $name => $class) {
-            if ('Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle' === $class) {
+            if (is_a($class, 'Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle', true)) {
```

Comment:
The third method argument is set to `true` in my code
> **allow_string**
If this parameter set to FALSE, string class name as object is not allowed. This also prevents from calling autoloader if the class doesn't exist.